### DISCUSSION TITLE
[Tutorial Vid] Independent Screen Resolution Rendering using Rendertarget2D

### DISCUSSION THREAD

**Discussion by AristurtleDev** [MonoGame/MonoGame#8189] [Graphics]
[![Independent Screen Resolution Rendering using RenderTarget2D](https://img.youtube.com/vi/Zla4q0Z6Zwc/0.jpg)](https://www.youtube.com/watch?v=Zla4q0Z6Zwc)

https://youtu.be/Zla4q0Z6Zwc?si=ARjZH6SPb4uEcPBZ

Topic came up in the [MonoGame Discord](https://discord.gg/monogame) earlier so thought it would make a good topic for a new video.

I'm sure I've got some errors in here somewhere feel free to critique the hell of of it.

**Comment by ThomasFOG** [comment]
Great technique far anything pixel art. This is what we use on our games (ScourgeBringer, NeuroVoider).

**Comment by jeremyfa** [comment]
In addition to this technique, if you want to handle better resolutions that are not a multiple of your original pixel art resolution, you might want to render the final texture with a shader like the one in this article: https://colececil.io/blog/2017/scaling-pixel-art-without-destroying-it/

**Reply by AristurtleDev** [reply]
Hey, thanks for the info.  The article looks Unity specific, which translating the shader to HLSL wouldn't be to much work.  Would you say that it would be a similar result to using the jitter free shader https://www.youtube.com/watch?v=bfhKKSeHCKI.

This one was based on the Godot implementation done by @uheartbeast

**Reply by jeremyfa** [reply]
Yes, looks like the technique is similar! And yes the article I shared is showcasing an example shader for Unity, but the rest of the explanation is pretty much engine-agnostic. I actually used that reference for non-unity tech too.
